### DISCUSSION TITLE
TS0222  Luminosity | Humidity | Temperature sensor (Moes)

### DISCUSSION THREAD

**Discussion by freitas666** [Koenkk/zigbee2mqtt#7537] [Devices]
Hello

I spended a long time trying put this sensor in my Z2M but looks like does not work....

Could someone help me?

I  attached some information about that....

Thank you.

![Captura de Tela 2021-05-19 às 21 31 15](https://user-images.githubusercontent.com/2224558/118901415-9b75a000-b8e9-11eb-8fd5-463f2f02d9b0.png)

**Comment by FelipeDamasio** [comment]
Me too, same issue here. Running the latest version of HA and Z2M.

**Comment by danielo515** [comment]
I bought this device because it was in the list of supported devices 
https://www.zigbee2mqtt.io/devices/TS0222.html
but after linking to my controller it says that it's not supported despite detecting correctly the device name and showing the right image.
